static/app/utils/useHoverOverlay.tsx

```diff
@@ -266,14 +266,35 @@ function useHoverOverlay(
    */
   const wrapTrigger = useCallback(
     (triggerChildren: React.ReactNode) => {
-      const props = {
+      const makeProps = (
```

Comment:
import {mergeProps} from '@react-aria/utils'; https://github.com/adobe/react-spectrum/blob/main/packages/%40react-aria/utils/src/mergeProps.ts#L48-L54

Reply:
Oh sweet, I did not know this existed. Thank you!